src/probnum/randprocs/covfuncs/_covariance_function.py

```diff
@@ -9,9 +9,16 @@
 
 import numpy as np
 
+try:
+    from pykeops.numpy import LazyTensor, Pm, Vi, Vj
+except ImportError:
+    pass
+
```

Comment:
It might be a good idea to move these imports into `_keops_lazy_tensor`, not sure though